README.md

```diff
@@ -164,14 +165,16 @@ jobs:
         with:
           required-ros-distributions: ${{ matrix.ros_distribution }}
       - name: build and test
-        uses: ros-tooling/action-ros-ci@0.0.17
+        uses: ros-tooling/action-ros-ci@0.0.18
         with:
-          package-name: YOUR_PACKAGE_NAME
+          package-name: ${{ matrix.package }}
+          target-ros2-distro: ${{ matrix.ros_distribution }}
 
   test_docker: # On Linux, iterates on all ROS 1, and ROS 2 distributions.
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        package: [YOUR_PACKAGE, OR_PACKAGES]  # <-- Fill in the name of your package(s)
```

Comment:
you can build multiple packages from the repo in one go by providing multiple inputs e.g.

```
with:
  package-name: package1 package2
```

Is there a particular isolated use case you're trying to demonstrate via using the build matrix and going through the whole setup for each package?

Reply:
Ah, no I guess I just didn't realize I could pass in multiple packages in one go! That's awesome to hear - I'll try it out.